netmanager/src/views/components/AirQlouds/AirQloudDataReport.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(2),
+  },
+}));
+
+function AirQloudDataReport() {
+  const classes = useStyles();
+  const history = useHistory();
+  const dispatch = useDispatch();
+
+  return (
+    <Paper className={classes.root}>
+      <h2>Report Title</h2>
+      <p>This is the content of the report.</p>
+    </Paper>
+  );
+}
+
+export default AirQloudDataReport;

```

Comment:
Thanks @sserurich . The file called `AirQloudDataReport` doesn't seem to be used anywhere

Reply:
@Codebmk , I have removed this and made few updates, kindly  test this out.